src/main/java/com/KacFlor/ShopSpring/config/UserConfig.java

```diff
@@ -22,10 +24,13 @@ public UserConfig(){
     }
 
     @Bean
-    CommandLineRunner commandLineRunnerUser(UserRepository userRepository, CustomerRepository customerRepository){
+    CommandLineRunner commandLineRunnerUser(CartRepository cartRepository ,UserRepository userRepository, CustomerRepository customerRepository){
```

Comment:
Do you remember as I told you to make special directory for default data creation "developer helper"?

Reply:
done

Reply:
It is not done